resources/rooms/room_conv.py

```diff
@@ -0,0 +1,64 @@
+"""
+Combines Tabula's mapping and Timetable reports to form single room mapping
+
+Update central room info periodically from https://warwick.ac.uk/services/its/servicessupport/av/lecturerooms/roominformation/room-data.js
+  Convert from js to json, e.g. with https://www.convertsimple.com/convert-javascript-to-json/
+Update tabula mapping `tabula-sciencianame.txt"` from Tabula src code: common/src/main/scala/uk/ac/warwick/tabula/services/timetables/ScientiaCentrallyManagedRooms.scala
+  Chop off ends and comments, then can regex convert `"(.+)" -> MapLocation\("(.+)", "(\d+)", Some\("(.+)"\)\),` to `$2 | $1`
+Update scientia mapping `scientianame-url.txt` from http://go.warwick.ac.uk/timetablereports > Locations > Inspect "Select Room(s)" menu, and copy out
+  Again also regex convert
+"""
+
+
+def read_mapping(filename):
+    with open(str(filename)) as f:
+        l = [l.split(" | ") for l in f.readlines()]
+        return {x[0].strip(): x[1].strip() for x in l if len(x) > 1}
```

Comment:
Consider adding [frozendict](https://pypi.org/project/frozendict/) as a dependency and returning here to avoid possiblity of mutation.

Reply:
Not needed as this is a standalone script.